### DISCUSSION TITLE
How do you use Reach with other frontend frameworks that aren't React?

### DISCUSSION THREAD

**Discussion by cb0806151** [reach-sh/reach-lang#76] [Q&A] [answered]
I'm currently working on a tutorial showing how you can connect an Angular RPS web app to the blockchain using Reach. Is there a way to run the setup that is happening with `./reach react` without running a react application? Kind of a framework agnostic way of running it, so you can start a frontend elsewhere on your machine and connect to reach with the standard library

**Comment by DanBurton** [comment, marked as answer]
There is nothing particularly special about using the `@reach-sh/stdlib` JavaScript library.

`reach react` is a convenience tool for giving people a taste of web programming, where we try to help avoid the tedium of setting it all up. But a seasoned React programmer likely won't need the simplified approach of `reach react`.

The Reach standard library for JS operates based on certain defaults, but is also intended to be configurable. If you simply `reach compile` your `index.rsh` code into `build/index.main.mjs`, then you can import this JavaScript file, and import the `@reach-sh/stdlib` library, as you would any other JavaScript file or library.

**Reply by DanBurton** [reply]
Note that, as a convenience, `./reach react` runs your react program and also starts a devnet for you. If you want a command to _only_ run the reach devnet, you can use `./reach devnet`.

**Comment by pirovanomg** [comment]
Can we close out this ticket?

**Reply by pirovanomg** [reply]
@cb0806151?
